### DISCUSSION TITLE
How to manually set value of the Controller select field?

### DISCUSSION THREAD

**Discussion by pwe-dev** [react-hook-form/react-hook-form#3592] [Q&A] [answered]
Hello, I need to manually set the value of material-ui select, after fetching some data. I can't use setValue() because component is not registered via register but it is using Controller component and control prop. How do I change the value of this select?

**Comment by bluebill1049** [comment, marked as answer]
you can use setValue with Controller

**Reply by pwe-dev** [reply]
I don't know why it didn't work for me earlier. Nvm, now it is working. Thanks

**Reply by bluebill1049** [reply]
cool.
